app/gui2/src/util/database/reactiveDb.ts

```diff
@@ -25,11 +26,15 @@ export type OnDelete = (cleanupFn: () => void) => void
  * @typeParam K - The key type for the database entries.
  * @typeParam V - The value type for the database entries.
  */
-export class ReactiveDb<K, V> extends ObservableV2<{
-  entryAdded(key: K, value: V, onDelete: OnDelete): void
-}> {
+export class ReactiveDb<K, V>
+  extends ObservableV2<{
+    entryAdded(key: K, value: V, onDelete: OnDelete): void
+  }>
+  implements Iterable<[K, V]>
+{
```

Comment:
This got a bit long now. Move the message type (object with `entryAdded`) to a separate type for clarity.